src/gui/widgets/editor_settings_page.cpp

```diff
@@ -44,6 +45,12 @@ EditorSettingsPage::EditorSettingsPage(QWidget* parent)
 {
 	auto layout = new QFormLayout(this);
 	
+	if (MainWindow::mobileMode())
+	{
+		button_size = Util::SpinBox::create(1, 3.0, 26.0, tr("mm", "millimeters"), 0.1);
```

Comment:
@lpechacek How about making this an integer, similar to the following 'Symbol icon size'? It is always a little bit difficult to edit this setting. At least I would like to change the step size from 0.1 to 0.5.